docker/scripts/ci_start.sh

```diff
@@ -0,0 +1,313 @@
+    info "Starting mounting map volumes ..."
+    if [ -n "${USER_SPECIFIED_MAP}" ]; then
+        for map_name in ${USER_SPECIFIED_MAP}; do
+            restart_map_volume_if_needed "${map_name}" "${VOLUME_VERSION}"
+        done
+    fi
+
+    if [ "$FAST_MODE" = "no" ]; then
+        for map_name in ${DEFAULT_MAPS[@]}; do
+            restart_map_volume_if_needed ${map_name} "${VOLUME_VERSION}"
+        done
+    else
+        for map_name in ${DEFAULT_TEST_MAPS[@]}; do
+            restart_map_volume_if_needed "${map_name}" "${VOLUME_VERSION}"
+        done
+    fi
+}
+
+function mount_other_volumes() {
+    info "Mount other volumes ..."
+    local volume_conf=
+    if [ "${FAST_MODE}" = "no" ]; then
+        # YOLO3D
+        local yolo3d_volume="apollo_yolo3d_volume_${USER}"
+        local yolo3d_image="${DOCKER_REPO}:yolo3d_volume-${TARGET_ARCH}-latest"
+        volume_conf="${volume_conf} --volumes-from ${yolo3d_volume}"
+    fi
+
+    # LOCALIZATION
+    local localization_volume="apollo_localization_volume_${USER}"
+    local localization_image="${DOCKER_REPO}:localization_volume-${TARGET_ARCH}-latest"
+    volume_conf="${volume_conf} --volumes-from ${localization_volume}"
+
+    if [ "${TARGET_ARCH}" = "x86_64" ]; then
+        local local_3rdparty_volume="apollo_local_third_party_volume_${USER}"
+        local local_3rdparty_image="${DOCKER_REPO}:local_third_party_volume-${TARGET_ARCH}-latest"
+        volume_conf="${volume_conf} --volumes-from ${local_3rdparty_volume}"
+    fi
+
+    OTHER_VOLUME_CONF="${volume_conf}"
+}
+
+function main() {
+    check_host_environment
+
+    parse_arguments "$@"
+
+    determine_dev_image "${USER_VERSION_OPT}"
+
+    if [ "${USE_LOCAL_IMAGE}" = "yes" ];then
+        info "Start docker container based on local image : ${APOLLO_DEV_IMAGE}"
+    fi
+
+    if ! docker_pull "${APOLLO_DEV_IMAGE}" ; then
+        error "Failed to pull docker image."
+        exit 1
+    fi
+
+    info "Determine whether host GPU is available ..."
+    determine_gpu_use_host
+    info "USE_GPU_HOST: ${USE_GPU_HOST}"
+    local local_host="$(hostname)"
+    local local_volumes=
+    setup_devices_and_mount_local_volumes local_volumes
+    mount_map_volumes
+    mount_other_volumes
+    set -x
+
+    ${DOCKER_RUN} -it --rm   \
+        --privileged    \
+        -e NVIDIA_VISIBLE_DEVICES=all \
+        -e NVIDIA_DRIVER_CAPABILITIES=compute,video,graphics,utility \
+        ${MAP_VOLUME_CONF}      \
+        ${OTHER_VOLUME_CONF}    \
+        ${local_volumes}        \
+        --net host \
+        -w /apollo \
+        --add-host "${DEV_INSIDE}:127.0.0.1" \
+        --add-host "${local_host}:127.0.0.1" \
+        --hostname "${DEV_INSIDE}" \
+        --shm-size 2G   \
+        --pid=host      \
+        -v /dev/null:/dev/raw1394 \
+        "${APOLLO_DEV_IMAGE}" \
+        /bin/bash /apollo/scripts/apollo_ci.sh
+
+    if [ $? -ne 0 ];then
+        error "Failed to start docker container  based on image: ${APOLLO_DEV_IMAGE}"
```

Comment:
Since this is a one-time container used for CI only, helper msg should be something like "Failed to start/run CI docker based on..." .